thirdeye/thirdeye-frontend/app/helpers/get-safe.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export function getSafe([dict, key]) {
```

Comment:
Would love to see some sort of documentation here i.e. why this custom helper is needed

Reply:
fixed